addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/ExtensionAjax.java

```diff
@@ -91,10 +96,51 @@ public ExtensionAjax() throws ClassNotFoundException {
     public void init() {
         super.init();
 
+        // Prevent Crawljax from logging too many, not so useful, INFO/WARN messages.
+        setLogLevel(
+                List.of(
+                        "com.crawljax.core.Crawler",
+                        "com.crawljax.core.CrawlTaskConsumer",
+                        "com.crawljax.core.CrawlSession",
+                        "com.crawljax.core.UnfiredCandidateActions",
+                        "com.crawljax.core.UnfiredFragmentCandidates",
+                        "com.crawljax.core.plugin.Plugins",
+                        "com.crawljax.core.state.InMemoryStateFlowGraph",
+                        "com.crawljax.core.state.StateMachine",
+                        "com.crawljax.forms.FormHandler",
+                        "io.github.bonigarcia.wdm.WebDriverManager",
+                        "org.openqa.selenium.devtools.CdpVersionFinder"),
+                Level.ERROR);
+        // Logs an error each time Crawljax runs.
+        setLogLevel(List.of("com.crawljax.fragmentation.FragmentationPlugin"), Level.OFF);
```

Comment:
```suggestion
        // Disabled because it logs an error each time Crawljax runs.
        setLogLevel(List.of("com.crawljax.fragmentation.FragmentationPlugin"), Level.OFF);
```

Or `Set OFF`?

Reply:
Done.